vassal-doc/src/main/readme-referencemanual/ReferenceManual/GameModule.adoc

```diff
@@ -266,6 +265,30 @@ Applies to <<GamePiece.adoc#top,Game Pieces>> on all <<Map.adoc#top,Map Windows>
 |image:images/GlobalKeyCommand.png[]
 |===
 
+
+[#StartupGlobalKeyCommand]
+==== Startup Global Key Command
+
+Can print a welcome message, or perform some other task that needs to be done whenever the module is started up.
+
+image:images/StartupGlobalKeyCommand.png[]
+
+An extension of <<#GlobalKeyCommand,Global Key Command>> that fires automatically upon completion of module load, once all the key listeners are started up.
+All fields behave identically to the corresponding ones in <<#GlobalKeyCommand,Global Key Command>>, except that those pertaining to the physical representation of a Toolbar button are suppressed as being inapplicable.
+
+As of VASSAL 3.6, multiple start-up global key commands _can_ be depended on to run in the order they are listed in the module.
```

Comment:
Should that be 3.7, not 3.6?

Reply:
No, 3.6 is correct. Much of this particular document edit is merely moving the whole clump from the wrong place it was to the right place it goes.